src/atomate2/vasp/flows/mp.py

```diff
@@ -185,3 +190,71 @@ def make(self, structure: Structure, prev_dir: str | Path | None = None) -> Flow
             jobs += [static_job]
 
         return Flow(jobs=jobs, output=output, name=self.name)
+
+
+# update potcars to 54, use correct W potcar
+# use staticmaker for compatibility
+
+
+class MPVaspLobsterMaker(VaspLobsterMaker):
+    """
+    Maker to perform a Lobster computation.
+
+    The calculations performed are:
+
+    1. Optional optimization.
+    2. Static calculation with ISYM=0.
+    3. Several Lobster computations testing several basis sets are performed.
+
+    .. Note::
+
+        The basis sets can only be changed with yaml files.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    relax_maker : .BaseVaspMaker or None
+        A maker to perform a relaxation on the bulk. Set to ``None`` to skip the
+        bulk relaxation.
+    lobster_static_maker : .BaseVaspMaker
+        A maker to perform the computation of the wavefunction before the static run.
+        Cannot be skipped. It can be LOBSTERUNIFORM or LobsterStaticMaker()
+    lobster_maker : .LobsterMaker
+        A maker to perform the Lobster run.
+    delete_wavecars : bool
+        If true, all WAVECARs will be deleted after the run.
+    address_min_basis : str
+        A path to a yaml file including basis set information.
+    address_max_basis : str
+       A path to a yaml file including basis set information.
+    """
+
+    name: str = "lobster"
+    relax_maker: BaseVaspMaker | None = field(
+        default_factory=lambda: MPGGADoubleRelaxMaker()
+    )
+    lobster_static_maker: BaseVaspMaker = field(
+        default_factory=lambda: MPGGAStaticMaker(
+            input_set_generator=MPGGAStaticSetGenerator(
+                user_potcar_functional="PBE_54",
+                user_potcar_settings={"W": "W_sv"},
+                user_kpoints_settings={"reciprocal_density": 310},
+                user_incar_settings={
+                    "EDIFF": 1e-6,
+                    "NSW": 0,
+                    "LWAVE": True,
+                    "ISYM": 0,
+                    "IBRION": -1,
+                    "ISMEAR": -5,
+                    "LORBIT": 11,
+                    # "ICHARG": 0, # is this okay?
```

Comment:
It would be good to test both runs with and without structure optimization. There could be differences.

Reply:
Sure will test these :smiley: